### ISSUE TITLE
SSL Certificate not valid

### ISSUE THREAD

**Issue by mrmadhat** [Varying-Vagrant-Vagrants/VVV#1672]
## Expected Behavior

A valid SSL certificate when visiting a site e.g http://one.wordpress.test 

## Current Behavior

The certificate is showing as invalid

<img width="901" alt="screen shot 2018-11-02 at 21 26 40" src="https://user-images.githubusercontent.com/19492281/47941461-26d2e580-dee6-11e8-8831-63dcdd00f941.png">

I have set SSL to "Always trust" as per [Trusting the CA docs](https://varyingvagrantvagrants.org/docs/en-US/references/https/trusting-ca/)

![screen shot 2018-11-02 at 21 28 41](https://user-images.githubusercontent.com/19492281/47941530-68639080-dee6-11e8-989b-3eae4b6b3655.png)

## Steps to Reproduce (for bugs)

1. Install VVV by following the documentation
2. Copy vvv-config.yml to vvv-custom.yml which includes tls-ca under utilities
3. Run `vagrant reload --provision`

## Your Environment

* VVV version: 2.4.0
* VVV Git Branch: master and develop
* Vagrant version: 2.2.0
* VM Provider name: Virtual Box
* VM Provider version: 5.2.20
* Operating System and version: MacOs Sierra 10.12.6

**Comment by mrmadhat** [comment]
I didn't reboot initially but I have now and it's still showing an invalid certificate

**Comment by tomjn** [comment]
What stories be is that it says the certificate is from avast, not VVV, did
you try to generate your own certificate manually?

**Comment by mrmadhat** [comment]
No, I had vvv setup previously but didn't use SSL. All I've done is update the programs required for vvv e.g vagrant and set the vvv cert to always trust. I do have avast installed that could possibly be something to do with it? I will try without Avast running and see if that has any effect.

**Comment by tomjn** [comment]
That's definitely related, I've no idea why `Avast` is showing as the certificate authority, it should be VVV internal

**Comment by tomjn** [comment]
Notice the issuing CA is VVV internal and matches the root certificate added. Yours says Avast. I don't know where that certificate came from or how it ended up there, but it is not a VVV certificate

<img width="605" alt="screenshot 2018-11-04 at 15 18 30" src="https://user-images.githubusercontent.com/58855/47965964-ef2e7f80-e044-11e8-8cc5-dc0d1f2eaf7b.png">

**Comment by mrmadhat** [comment]
After turning avast off the certificate shows as valid in chrome so that must be it. The certificate still show as invalid on firefox but I've added an exception so it's not that big of a deal. Thanks for your help 😄
